website/content/en/highlights/2021-11-12-event-trottle-transform.md

```diff
@@ -0,0 +1,28 @@
+Event trottle transform released
+
+We've released a new transform that provides a user the ability to control the throughput of specific event streams.
+
+Large spikes in data volume can frequently overwhelm a service, which is especially common in log data. Previously, users lacked the necessary tooling to control throughput, such as setting quota for users and user groups utilizing Vector, which not only can cause spike in costs, but also negatively impact downstream services due to the increased load. 
+
+`Throttle` transform enables you to rate limit specific subsets of your event stream to limit load on downstream services or enforce quotas on users. You can utilize 'throttle' transform to enforce rate limits on number of events and exclude events based on a VRL condition to avoid dropping critical logs. 
```

Comment:
```suggestion
`Throttle` transform enables you to rate limit specific subsets of your event stream to limit load on downstream services or enforce quotas on users. You can utilize the `throttle` transform to enforce rate limits on number of events and exclude events based on a VRL condition to avoid dropping critical logs. 
```